server/market/market.go

```diff
@@ -2113,3 +2177,31 @@ func (m *Market) orderResponse(oRecord *orderRecord) (*msgjson.Message, error) {
 	// Encode the order response as a message for the client.
 	return msgjson.NewResponse(oRecord.msgID, res, nil)
 }
+
+// SetFeeRateScale sets a swap fee scale factor for the given asset.
+// SetFeeRateScale should be called regardless of whether the Market is
+// suspended.
+func (m *Market) SetFeeRateScale(assetID uint32, scale float64) {
+	m.feeScalesMtx.Lock()
+	m.feeScales[assetID] = scale
```

Comment:
I think I might have `feeScales [2]float64` or even `baseFeeScale, quoteFeeScale uint32` instead of a map, but it's not unpalatable as a map.